sources/packages/web/src/components/common/IdleTimeChecker.vue

```diff
@@ -6,19 +6,33 @@
     @keyup="setLastActivityTime"
   >
     <slot></slot>
-    <ConfirmExtendTime ref="extendTimeModal" :clientIdType="clientIdType" />
+    <ConfirmExtendTime
```

Comment:
During a local test, once the modal is visible if the user clicked anywhere on the screen the modal is closed and the timer is reset. Is it the expected behavior?
I was expecting that the user must answer the modal Yes/No question. Can you try to replicate this?

Reply:
Yes, I thought that was the wished behavior because that is how the ModalDialogBase.vue works. That requirement is not clear. I can change that to wait for a yes/no answer.

Reply:
True, but it is also configurable. Checking the other modals in the solution they do not follow this behavior 😉 
I guess that there was a decision to force the user to pick one option.

Reply:
ok. I'll change that. Thanks!